### ISSUE TITLE
Contributing.md link is broken

### ISSUE THREAD

**Issue by Evavic44** [opensourcecommunity-hub/amazing-github-projects-collection#8]
Current link and the fix.
```diff
- https://github.com/opensourcecommunity-hub/amazing-github-projects-collection/blob/main/contributing.md
+ https://github.com/Evavic44/amazing-github-projects-collection/blob/main/CONTRIBUTING.md
```

**Comment by Susmita-Dey** [comment]
Okay, I'm assigning it to you. Go and fix it.

**Comment by Susmita-Dey** [comment]
@Evavic44 Are you still working on this issue?

**Comment by Evavic44** [comment]
> @Evavic44 Are you still working on this issue?

I wanted to reserve it for first-timers only. If it's cool, I can still go ahead.

**Comment by Susmita-Dey** [comment]
> > @Evavic44 Are you still working on this issue?
> 
> I wanted to reserve it for first-timers only. If it's cool, I can still go ahead.

It's been assigned to you. So, go ahead.

**Comment by Susmita-Dey** [comment]
@Evavic44 You didn't create any PR here in this repository.

**Comment by Evavic44** [comment]
My bad. I was doing that with my phone. Raising a PR now.

**Comment by Susmita-Dey** [comment]
Issue is closed by #9
